Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1654,6 +1654,26 @@ Messages::WebDriverClient::DeleteAllCookiesResponse WebDriverConnection::delete_
     return JsonValue {};
 }
 
+// 15.8 Release Actions, https://www.w3.org/TR/webdriver/#dfn-release-actions
```

Comment:
Not sure where this spec link came from, but should be:
`https://www.w3.org/TR/webdriver/#release-actions`
which links to the `<section>` that defines this (and is what we link everywhere else), rather a `<h3>` element inside of it.

Reply:
Not sure what you mean, exactly. The URL is wrong for sure. it should be

`https://w3c.github.io/webdriver` not `https://www.w3.org/TR/webdriver/`

But out of the 75 spec links in this file, 59 of them link to `webdriver/#dfn-something`

Reply:
Huh, weird, ignore this then. 

I grab the links from the little anchor symbol next to the header:
![Screenshot_20230821_140856](https://github.com/SerenityOS/serenity/assets/5600524/e588d46c-fc3a-4538-958a-23de87fbc31a)

which doesn't include `dfn-`, but ah well.